web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -92,6 +97,13 @@ private void onException(Exception e) {
      */
     public void init(FilterConfig config){
         try {
+            if (!ClassDataUtils.INSTANCE.isMethodExists(javax.servlet.http.HttpServletResponse.class, "getStatus")) {
+                InternalLogger.INSTANCE.error(
```

Comment:
Shouldn't this log also appear in the appserver logs, i.e. should be logged always?